File: server/src/main/java/cn/iinti/atom/entity/metric/MetricMinute.java
package cn.iinti.atom.entity.metric;

import com.baomidou.mybatisplus.annotation.TableName;
import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;

/**
 * <p>
 * 监控指标,分钟级
 * </p>
 *
 * @author iinti
 * @since 2023-03-13
 */
@Data
@EqualsAndHashCode(callSuper = false)
@Accessors(chain = true)
@TableName("metric_minute")
@Schema(name = "Metric对象", description = "监控指标")
public class MetricMinute extends Metric {
}

File: server/src/main/kotlin/cn/iinti/atom/entity/metric/MetricMinute.kt
package cn.iinti.atom.entity.metric

import com.baomidou.mybatisplus.annotation.TableName
import io.swagger.v3.oas.annotations.media.Schema
import lombok.Data
import lombok.EqualsAndHashCode
import lombok.experimental.Accessors

@Schema(name = "Metric对象", description = "监控指标,分钟级")
@TableName("metric_minute")
@Accessors(chain = true)
@Data
@EqualsAndHashCode(callSuper = false)
class MetricMinute : Metric() {

}